### ISSUE TITLE
Documentation on Determination

### ISSUE THREAD

**Issue by etanb** [CoreyGinnivan/whocanuse#63]
Hi there,

My name is Etan and I help work on accessibility work across the web, most recently at the CDC and currently at the VA.

We reference WhoCanUse quite a bit in our work! And we love it. But one thing that's missing for our reference is specifically _how_ each contrast ratio is calculated. Having that as a reference on the site or on a separate documentation site would be incredible for our work. I know you mention the libraries Chroma.js and color-blind  for some of the more contrast ratios. But for cataracts, glaucoma, low vision and situational events it would be great to see the behind-the-scenes calculation.
